### PR TITLE
Update NVIDIA drivers document with known issues and workarounds

### PR DESCRIPTION
**Pull request by puneetse**
This PR adds a section for known issues and workarounds to the NVIDIA drivers document.
Note: an image has been added which does not render on the GitHub preview.

It is also meant to address https://github.com/clearlinux/clear-linux-documentation/issues/919

closes https://github.com/clearlinux/clear-linux-documentation/issues/919

### CODE REVIEW

**Comment by puneetse** [comment]
@lebensterben fyi

File: source/tutorials/nvidia.rst [hunk 1]
```diff
@@ -320,6 +393,41 @@ Troubleshooting
...
+  about kernel module builds which can be useful if the NVIDIA driver breaks
+  between kernel upgrades.
+
+
+No display or blank screen
+==========================
+
+Check if the has come up on another graphics device, including the integrated
+graphics, if you see a black screen or the login screen has not come up after
+installing the NVIDIA drivers. This can happen before an Xorg config has been
```
Comment (bwarden) [inline source/tutorials/nvidia.rst]: The first sentence is really confusing.

File: source/tutorials/nvidia.rst [hunk 2]
```diff
@@ -320,6 +393,41 @@ Troubleshooting
...
+   :align: center
+
+   NVIDIA driver GNOME crash dialogue on Clear Linux OS.
+
+There have been reports of GNOME crashing with an "Oh no! Something has gone
+wrong" error message with NVIDIA drivers installed while other graphics
+devices are enabled.
+
+Try disabling other graphics devices, including integrated graphics, in your
+systems EFI/BIOS.
```
Comment (bwarden) [inline source/tutorials/nvidia.rst]:
```suggestion
system's EFI/BIOS. 
```

**Comment by lebensterben** [comment]
>@lebensterben fyi

@puneetse thanks, looks great

File: source/tutorials/nvidia.rst (hunk 1 above)
Comment (puneetse) [reply source/tutorials/nvidia.rst]: Yeah, yikes. Does my second attempt read better?

File: source/tutorials/nvidia.rst [hunk 3]
```diff
@@ -320,6 +393,43 @@ Troubleshooting
...
+  between kernel upgrades.
+
+
+No display or blank screen
+==========================
+
+Check to see if the display has come up on another graphics device, including
+the integrated graphics device.
+
+You can might get a black screen or the login screen might not come up after
```
Comment (bwarden) [inline source/tutorials/nvidia.rst]:
```suggestion
You might get a black screen or the login screen might not come up after
```

File: source/tutorials/nvidia.rst [hunk 4]
```diff
@@ -320,6 +393,43 @@ Troubleshooting
...
+installing the NVIDIA drivers before an Xorg configuration has been defined
```
Comment (bwarden) [inline source/tutorials/nvidia.rst]:
```suggestion
installing the NVIDIA drivers until an Xorg configuration has been defined
```

**Comment by mvincerx** [comment]
> This PR adds a section for known issues and workarounds to the NVIDIA drivers document.
> Note: an image has been added which does not render on the GitHub preview.
> 
> It is also meant to address #919

@puneetse, Do you want to close #919, as a result? I'm checking to assure issue scope is satisfied.

**Comment by puneetse** [comment]
Yes, thanks for checking. I added the keyword to the first post.

**Comment by mvincerx** [comment]
LGTM

**Pull request merged**